### ISSUE TITLE
In register form : remove mandatory effect for the fields phone and address

### ISSUE THREAD

**Issue by MatSarr** [Lokavaluto/lokavaluto-addons#66]
Some person won't take a online membership because it's mandatory to give the phone number and the address

In register form it would be great to remove the mandatory effect for those fields

**Comment by njeudy** [comment]
@MatSarr is it a need for all LCC ?

**Comment by MatSarr** [comment]
Yes! The mandatory fields should be only the name (it would be great to show first name et last name) and the email.

**Comment by njeudy** [comment]
for first name and last name on form you can use: `website_sale_firstname`

For the phone, is this an ask for all lcc ? or need I to create a small module for that ?

**Comment by MatSarr** [comment]
This is an old request. I guess this should be integrate in the global thought about the subscription form
